tests/xcm-transfer/astar-asset-hub.test.ts

```diff
@@ -31,8 +31,8 @@ describe('Astar & AssetHub', () => {
 
       const bobBalance = await query.assets(astarUtil.usdt)(astar, bob.address)
       expect(bobBalance.unwrap().balance.toNumber()).closeTo(
-        10e6,
-        1e5, // some fee
+        10_000_000,
```

Comment:
Why not adjusting `10e6` in the XCM txn also?

Reply:
Fee changed for usdt and I only touched that test